.github/workflows/codeql.yml

```diff
@@ -65,7 +78,10 @@ jobs:
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
       uses: github/codeql-action/autobuild@v2
-      if: ${{ matrix.language != 'java' }}
+      with:
+        working-directory: ${{ matrix.target.directory }}
```

Comment:
this is to make sure that autobuild will start in the correct subdirectory to try to build the code.

Reply:
Is there an instance where that's not the case?

Reply:
[In a previous run when autobuild was executed from the root of the directory](https://github.com/open-telemetry/opentelemetry-lambda/actions/runs/5930233093/job/16079525511#step:4:50), it tried to run the build using an unrelated python script for the case it was running for the cshar language. Using a different working directory will ensure that autobuild will only find build scripts related to the language that is set.